hogben/simulate.py

```diff
@@ -218,15 +238,16 @@ def _run_experiment(sample, angle, points, time, scale, bkg, dq,
     # Point has zero reflectivity if there is no flux.
     r_noisy = np.divide(counts_reflected, counts_incident,
                         out=np.zeros_like(counts_reflected),
-                        where=counts_incident!=0)
+                        where=counts_incident != 0)
 
     r_error = np.divide(np.sqrt(counts_reflected), counts_incident,
                         out=np.zeros_like(counts_reflected),
-                        where=counts_incident!=0)
+                        where=counts_incident != 0)
 
     return q_binned, r_noisy, r_error, counts_incident
 
-def reflectivity(q, model):
+
+def reflectivity(q: np.ndarray, model) -> np.ndarray:
```

Comment:
np.ndarray can now be removed from the docstring. Although a description of the type/shape of the array could be useful instead.